### PR TITLE
fix hugegraph-api code checkstyle

### PR DESCRIPTION
**Pull request by Fizell**
rt

### CODE REVIEW

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java [hunk 1]
```diff
@@ -155,7 +151,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
         private void sendError(ChannelHandlerContext ctx, Object msg) {
             // Close the connection as soon as the error message is sent.
             ctx.writeAndFlush(new DefaultFullHttpResponse(HTTP_1_1, UNAUTHORIZED))
-               .addListener(ChannelFutureListener.CLOSE);
+                .addListener(ChannelFutureListener.CLOSE);
```
Comment (imbajin) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java:154]: maybe we should keep the origin align?

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java [hunk 2]
```diff
@@ -114,19 +113,16 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
...
-                    sendError(ctx, msg);
-                    return;
-                } catch (IllegalArgumentException iae) {
+                } catch (IndexOutOfBoundsException | IllegalArgumentException iae) {
                     sendError(ctx, msg);
                     return;
                 }
                 String authorization = new String(userPass,
-                                                  Charset.forName("UTF-8"));
+                        StandardCharsets.UTF_8);
```
Comment (imbajin) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java:125]: we use the `hugegraph-style` in the wiki (not java original style)

align the param with the highly priority

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java [hunk 3]
```diff
@@ -45,6 +27,23 @@
...
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.netty.handler.codec.http.HttpHeaderNames.CONNECTION;
+import static io.netty.handler.codec.http.HttpHeaderNames.UPGRADE;
+import static io.netty.handler.codec.http.HttpResponseStatus.UNAUTHORIZED;
+import static io.netty.handler.codec.http.HttpVersion.HTTP_1_1;
+import static org.apache.tinkerpop.gremlin.groovy.jsr223.dsl.credential.CredentialGraphTokens.PROPERTY_PASSWORD;
+import static org.apache.tinkerpop.gremlin.groovy.jsr223.dsl.credential.CredentialGraphTokens.PROPERTY_USERNAME;
```
Comment (imbajin) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java:46]: so as these import rules, u could refer the [style file](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-style.xml) in root dir to improve them

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/StandardAuthenticator.java [hunk 4]
```diff
@@ -95,7 +99,7 @@ private String inputPassword() {
             if (!password.isEmpty()) {
                 return password;
             }
-            System.out.println(notEmptyPrompt);
+            LOG.info(notEmptyPrompt);
```
Comment (imbajin) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/StandardAuthenticator.java:102]: seems it was not print in the log file (print on the console is right?) @javeme

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java [hunk 5]
```diff
@@ -1757,7 +1731,7 @@ public static class ContextThreadPoolExecutor extends ThreadPoolExecutor {
         public ContextThreadPoolExecutor(int corePoolSize, int maxPoolSize,
                                          ThreadFactory threadFactory) {
             super(corePoolSize, maxPoolSize, 0L, TimeUnit.MILLISECONDS,
-                  new LinkedBlockingQueue<Runnable>(), threadFactory);
+                    new LinkedBlockingQueue<>(), threadFactory);
```
Comment (imbajin) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java:1734]: keep the origin align~

**Review by imbajin — COMMENTED** [review]
seems a lot of the code style conflict, maybe we could use a unified rule before change it

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java (hunk 1 above)
Comment (Fizell) [reply hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java:154]: but the current style check specify that the code indented in multiples of 4，so we need to decide which style to use

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java (hunk 5 above)
Comment (Fizell) [reply hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java:1734]: same as above

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java (hunk 1 above)
Comment (imbajin) [reply hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java:154]: > but the current style check specify that the code indented in multiples of 4，so we need to decide which style to use

yep, we'll discuss this issue in the developer group, ensure the rule

**Comment by imbajin** [comment]
Open a new vote discussion in [#1848](https://github.com/apache/incubator-hugegraph/discussions/1848)

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeAuthenticator.java [hunk 6]
```diff
@@ -254,7 +248,7 @@ public static class UserJson {
         }
     }
 
-    public static class RolePerm {
+    class RolePerm {
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeAuthenticator.java:251]: prefer to keep `public static class`

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeAuthenticator.java [hunk 7]
```diff
@@ -394,7 +388,7 @@ public static boolean match(Object role, RolePermission grant,
         }
     }
 
-    public static class RequiredPerm {
+    class RequiredPerm {
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeAuthenticator.java:391]: prefer to keep `public static class`

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeFactoryAuthProxy.java [hunk 8]
```diff
@@ -60,17 +61,20 @@
...
 
+    private static final Logger LOG = Log.logger(HugeFactoryAuthProxy.class);
+
     public static final String GRAPH_FACTORY =
-           "gremlin.graph=com.baidu.hugegraph.auth.HugeFactoryAuthProxy";
+            "gremlin.graph=com.baidu.hugegraph.auth.HugeFactoryAuthProxy";
 
     private static final Set<String> PROTECT_METHODS = ImmutableSet.of(
-                                                       "instance");
+            "instance");
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeFactoryAuthProxy.java:75]: align with ImmutableSet

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java [hunk 9]
```diff
@@ -210,16 +210,12 @@ public Collection<PropertyKey> propertyKeys() {
 
     @Override
     public PropertyKey propertyKey(String key) {
-        return verifySchemaPermission(HugePermission.READ, () -> {
-            return this.hugegraph.propertyKey(key);
-        });
+        return verifySchemaPermission(HugePermission.READ, () -> this.hugegraph.propertyKey(key));
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java:213]: prefer to keep origin style for debug and readability

File: hugegraph-api/src/main/java/com/baidu/hugegraph/auth/StandardAuthenticator.java (hunk 4 above)
Comment (javeme) [reply hugegraph-api/src/main/java/com/baidu/hugegraph/auth/StandardAuthenticator.java:102]: yes we need System.out sometimes like when using the console

**Review by javeme — CHANGES_REQUESTED** [review]

**Comment by Fizell** [comment]
seems that tends to keep the original align and keep original lambda style, so I submit a new PR to fix only some obvious style issues https://github.com/apache/incubator-hugegraph/pull/1851, and this PR will be closed @javeme @imbajin

**Pull request closed without merging**